src/devices/uFireIse/README.md

```diff
@@ -0,0 +1,15 @@
+﻿# μFire ISE Probe Interface - I2C Driver
+
+## Summary
+
+The μFire ISE Probe Interface is an I²C sensor that can read a pH probe. Attach a waterproof temperature sensor for temperature compensation with the attached connector.
+
```

Comment:
Usually we ask to add the usage of the device in the bindings. How to create the device, basic usage for reading, things like this.